model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -32,6 +32,20 @@ option java_outer_classname = "RunnerApi";
 import "google/protobuf/any.proto";
 import "google/protobuf/descriptor.proto";
 
+message BeamConstants {
+  enum Constants {
+    // The maximum timestamp for the global window in milliseconds since Jan 1, 1970.
+    // Triggers use maxTimestamp to set timers' timestamp. Timers fires when
+    // the watermark passes their timestamps. So, the timestamp needs to be
+    // smaller than the TIMESTAMP_MAX_MILLIS.
+    // One standard day is subtracted from TIMESTAMP_MAX_VALUE to make sure
+    // the maxTimestamp is smaller than TIMESTAMP_MAX_VALUE even after rounding up
+    // to seconds or minutes. See also GlobalWindow in the Java SDK.
+    // Value is represented in hex.
+    GLOBAL_WINDOW_MAX_TIMESTAMP_MILLIS = 0 [(beam_constant) = "20c49ba0bcf7f7"];
```

Comment:
@robertwb Hex doesn't seemed to have helped the readability here, can we stick with a number base 10?

Reply:
Yes, my other comment seems to have been lost, but I was hoping for a nice round 0xFFFFF..... (and also suggested adding TIMESTAMP_MAX_MILLIS).